### ISSUE TITLE
Clarification

### ISSUE THREAD

**Issue by taylor-bsg** [T-head-Semi/openc910#9]
Is the source code generated from some other source language? We notice the gen_RTL directory name.

This would be useful to know; for example, will pull requests to this repo be accepted, or would the fixes need to be made upstream and then the RTL regenerated by T Head?

**Comment by CaffreyCC** [comment]
Yes, the code is generated through an internal tool. And of course the project will accept PRs with changes to the RTL under appropriate conditions. We are planning to introduce a more complete verification environment to ensure that all the modifications to the project are correct in function and timing.

**Comment by robtaylor** [comment]
I would encourage the maintainers that they commit the original vperl code. Then at least there would be an option for modifying in the original source, and the community could potentially provide a replacement to the proprietary tool.
